This patch to QOwnNotes 26.9 adds/modifies a couple of keystrokes in
the editor:

 * Ctrl-A : beginning of line
 * Ctrl-E : end of line
 * NumPad 0 : beginning of next line

Because I'm used to it :).

*** qownnotes-26.9.6/libraries/qmarkdowntextedit/qmarkdowntextedit.cpp	2026-09-16 21:03:33.000000000 +0200
--- qownnotes-26.9.6-build/qownnotes-26.9.6/libraries/qmarkdowntextedit/qmarkdowntextedit.cpp	2026-09-17 16:04:16.445874529 +0200
***************
*** 497,502 ****
--- 497,523 ----
                     (keyEvent->modifiers().testFlag(Qt::ShiftModifier))) {
              moveTextUpDown(true);
              return true;
+         } else if ((keyEvent->key() == Qt::Key_A) &&
+                    (keyEvent->modifiers().testFlag(Qt::ControlModifier)) &&
+                    !(keyEvent->modifiers().testFlag(Qt::ShiftModifier))) {
+             QTextCursor cursor = textCursor();
+             cursor.movePosition(QTextCursor::StartOfLine);
+             this->setTextCursor(cursor);
+             return true;
+         } else if ((keyEvent->key() == Qt::Key_E) &&
+                    (keyEvent->modifiers().testFlag(Qt::ControlModifier)) &&
+                    !(keyEvent->modifiers().testFlag(Qt::ShiftModifier))) {
+             QTextCursor cursor = textCursor();
+             cursor.movePosition(QTextCursor::EndOfLine);
+             this->setTextCursor(cursor);
+             return true;
+         } else if ((keyEvent->key() == Qt::Key_0) &&
+                    (keyEvent->modifiers().testFlag(Qt::KeypadModifier))) {
+             QTextCursor cursor = textCursor();
+             cursor.movePosition(QTextCursor::StartOfLine);
+             cursor.movePosition(QTextCursor::Down);
+             this->setTextCursor(cursor);
+             return true;
  #ifdef Q_OS_MAC
              // https://github.com/pbek/QOwnNotes/issues/1593
              // https://github.com/pbek/QOwnNotes/issues/2643
